developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -47,35 +47,72 @@ const baseOptions: CompilerOptions = {
  */
 let callbackProcIdentifier = 0;
 
+/**
+ * The wrapped functions
+ */
+class WrappedWasmFuncs {
+  compileKeyboardFile?: (a0: string, a1: string, a2: number, a3: number, a4: number, a5: string) => boolean;
+  parseUnicodeSet?: (a0: string, a1: number, a2: number) => number;
```

Comment:
I think we should give the actual parameter names here for posterity -- grab them from the C++ code

Reply:
I'll put a reference in too, since I keep forgetting they are in CompilerInterfaces.cpp (for the _Wasm versions)